packages/default/scss/input/_layout.scss

```diff
@@ -56,6 +57,49 @@
             }
         }
 
+        .k-input-prefix,
+        .k-input-suffix {
+            display: flex;
+            align-items: center;
+            flex-shrink: 0;
+            height: calc( #{$button-line-height * 1em} + #{$button-padding-y * 2});
```

Comment:
we have a variable for that: it's `$button-inner-calc-size` https://github.com/telerik/kendo-themes/blob/6182e08e9cc2fecbd89f1739e3c32ac1ec83d50b/packages/default/scss/_variables.scss#L763-L764